docs/standard/microservices-architecture/net-core-net-framework-containers/general-guidance.md

```diff
@@ -25,15 +25,23 @@ In short, when you create new containerized .NET applications, you should consid
 
 An additional benefit of using .NET Core is that you can run side by side .NET versions for applications within the same machine. This benefit is more important for servers or VMs that do not use containers, because containers isolate the versions of .NET that the app needs. (As long as they are compatible with the underlying OS.)
 
-You should use .NET Framework for your containerized Docker server application when:
+You should use .NET Framework, with Windows Containers, for your containerized Docker server application when:
 
 -   Your application currently uses .NET Framework and has strong dependencies on Windows.
 
 -   You need to use Windows APIs that are not supported by .NET Core.
 
 -   You need to use third-party .NET libraries or NuGet packages that are not available for .NET Core.
 
-Using .NET Framework on Docker can improve your deployment experiences by minimizing deployment issues. This "lift and shift" scenario is important for "dockerizing" legacy applications (at least, those that are not based on microservices).
+Using .NET Framework on Docker can improve your deployment experiences by minimizing deployment issues. This [*"lift and shift" scenario*](https://aka.ms/liftandshiftwithcontainersebook) is important for "dockerizing" legacy applications that were originally developed with the traditional .NET Framework, like ASP.NET WebForms, MVC web apps or WCF (Windows Communication Foundation) services.
```

Comment:
nit: I usually see "containerizing" preferred to "dockerizing"

Reply:
👍  💯